code/style.css

```diff
@@ -37,16 +69,31 @@ p {
   border-radius: 16px;
 }
 
+.weapon-deck:hover {
+  cursor: pointer;
+  opacity: 0.8;
+}
+
```

Comment:
Is there any way you could do this for all decks with a single class?  Maybe something to discuss with your team?